### DISCUSSION TITLE
NFS Soft Mount

### DISCUSSION THREAD

**Discussion by benjamindaines** [borgbase/vorta#1683] [General]
What happens if a soft-mount NFS share gets disconnected during a backup?  Say I don't remember to check if Vorta / borg is running and I shut the lid while it's running.

**Comment by real-yfprojects** [comment]
A similar question came up before but I can't find it. AFAIK this might corrupt the created archive but shouldn't endanger any other data. Maybe @ThomasWaldmann can give further insight. 

@ThomasWaldmann since similar question came up a couple of times already, it might make sense to add this to the FAQ.

**Reply by ThomasWaldmann** [reply]
In general, a failed borg backup does not influence other (older) backup archives.

Borg just reads files and directories, without doing anything special, so whatever the OS / fs yields will end up in the newly created backup archive (and also: whatever is NOT visible will of course NOT end up in the new archive).

The user is responsible for presenting the files in a stable/consistent/complete way so they are "backup-worthy", e.g.:
- not backing up live databases or VMs (will likely have consistency issues then)
- not forgetting to mount before backing up the mountpoint (borg will happily backup an empty directory also)

**Reply by benjamindaines** [reply]
So lets say a backup is running and currently backing up new data, that archive is interrupted and corrupted.  Will those new files be backed up in the next archive, or will the system see them as previously backed up despite the archive they were backed up in being corrupt?

**Reply by ThomasWaldmann** [reply]
If it is interrupted (like power failure, fatal exception (rc 2), ssh connection breaking down), it does not save the files cache to disk, so it uses the previous files cache state for the next backup run (to determine what files are unchanged).
